src/components/entries/FEEL/Feel.js

```diff
@@ -622,7 +622,7 @@ export default function FeelEntry(props) {
     }
 
     setValidationError(newValidationError);
-  }, [ element ]);
+  }, [ element, getValue, setValue, validate ]);
```

Comment:
BTW this shows how all three text components could use shared logic. I don't want to bloat the changes today, so I'm leaving this as is.